WIP: Add acknowledgements section.

mickenordin opened the pull request:
I am making this a wip pr in case I have missed someone who should be mentioned.

mickenordin committed:
WIP: Add acknowledgements section.

I am making this a wip pr in case I have missed someone who should be
mentioned.

mickenordin commented:
I suppose we ought to mention Geant and NL-net as well? Some other organization?

glpatcern commented on IETF-RFC.md:
```diff
@@ -613,4 +613,14 @@ Following the validation of the signature, the host should also confirm the vali
+
+In addition we would like to thank Yashar PourMohammad, Parham Rahimpour, Navid Shokri and Reza Soltani for their contributions to the implementation of the OCM test suite.
+
+Over the years many more people have been involved in the development of OCM. We would like to thank all of them for their contributions, including Jean-Thomas Acquaviva, Samuel Alfageme Sainz, Karsten Asshauer, Miroslav Bauer, Felix Böhm, Maciej Brzeźniak, Diogo Castro, Gavin Charles Kennedy, Jarosław Czub, Milan Danecek, Michael D'Silva, Lukasz Dutka, Pedro Ferreira, Renato Furter, Klaas Freita ...
```
As P. Szegedi was mentioned before, is it worth mentioning it again?

mickenordin replied:
Good catch

glpatcern commented on IETF-RFC.xml:
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
   <?xml-stylesheet type="text/xsl" href="rfc2629.xslt" ?>
-  <!-- generated by https://github.com/cabo/kramdown-rfc version 1.7.29 (Ruby 3.2.3) -->
+  <!-- generated by https://github.com/cabo/kramdown-rfc version 1.7.29 (Ruby 3.3.8) -->
```
Mahdi just pushed a CI workflow to generate the xml file, you may want to drop this file.

glpatcern reviewed (approved):
@mickenordin I see you went through really the full history, definitely I don't have further names to add. Just a couple of comments:

glpatcern commented:
> I suppose we ought to mention Geant and NL-net as well? Some other organization?

For that, I'm not sure: we'd have to either mention the affiliation of everyone, or none of them.

MahdiBaghbani commented on IETF-RFC.md:
```diff
@@ -613,4 +613,14 @@ Following the validation of the signature, the host should also confirm the vali
+
+We would also like to thank Ishank Arora, Gianmaria Del Monte, Jörn Friedrich Dreyer, Hugo González Labrador, Maxence Lange, Lovisa Lugnegård, Sandro Mesterheide, Antoon Prins and Björn Schissle for their direct contributions to the specification.
+
+In addition we would like to thank Yashar PourMohammad, Parham Rahimpour, Navid Shokri and Reza Soltani for their contributions to the implementation of the OCM test suite.
```
Dev-Stock has been a repo we used for everything in the Ponder Source app development, so while the people here have commits in the dev-stock repo they didn't work on test suite,

Sandro & Giuseppe have contributed to the OCM Test Suite parts of the dev-stock

mickenordin replied:
ok, I will remove that part since Sandro is mentioned above and Giuseppe is an author.

mickenordin commented:
> > I suppose we ought to mention Geant and NL-net as well? Some other organization?
> 
> For that, I'm not sure: we'd have to either mention the affiliation of everyone, or none of them.

I was thinking that we should thank those two orgs specifically because of direct contribution of money, but I am happy to exclude such a section too.

mickenordin committed:
Address feedback

Remove duplicate name, remove testsuite section and drop the xml file
that now will be generated in ci.
